src/Symfony/Component/Validator/Resources/translations/validators.es.xlf

```diff
@@ -192,7 +192,7 @@
             </trans-unit>
             <trans-unit id="51">
                 <source>No temporary folder was configured in php.ini.</source>
-                <target>Ninguna carpeta temporal fue configurada en php.ini.</target>
+                <target>Ninguna carpeta temporal fue configurada en php.ini o la carpeta configurada no existe.</target>
             </trans-unit>
```

Comment:
I thought that it might be a good idea to adjust the Spanish translation at the same time. @javiereguiluz Could please have a look at this one?

Reply:
The translation is correct! Thanks.